burger-app/src/components/Header/Menu/index.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+const Menu = () => {
```

Comment:
It should be updated 

https://github.com/mariianazarova/React/pull/2#discussion_r1056866328